azure-pipelines.yml

```diff
@@ -11,14 +15,23 @@ variables:
   TRAVIS_PULL_REQUEST_SHA: $(Build.SourceVersion)
   PR_ONLY: 'true'
 
-jobs:
-- template: .azure-pipelines/BranchProtectionForPrivateRepo.yml
-- template: .azure-pipelines/Syntax.yml
-- template: .azure-pipelines/Semantic.yml
-- template: .azure-pipelines/Avocado.yml
-- template: .azure-pipelines/ModelValidation.yml
-- template: .azure-pipelines/BreakingChange.yml
-- template: .azure-pipelines/LintDiff.yml
-- template: .azure-pipelines/SwaggerToSDK.yml
-- template: .azure-pipelines/NetworkValidation.yml
-- template: .azure-pipelines/Spellcheck.yml
+stages:
```

Comment:
what's difference between stage and job

Reply:
stage are set of jobs that could be triggered together with certain condition or work flow.